specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -456,9 +579,13 @@ model PreDefinedConcept {
 
 @added(Versions.v2023_10_30_Preview)
 @doc("Label definition.")
-model SubCategory {
+@renamedFrom(Versions.v2024_01_30_Preview, "SubCategory")
+model Class {
```

Comment:
This model name won't be acceptable for SDK architects. `Class` is also a reserved word in many languages. Please consider a better name.

Reply:
Sure. Thanks for the information!